index.bs

```diff
@@ -787,6 +795,38 @@ option. If {{UnderlyingSource/type}} is set to undefined (including via omission
      |underlyingSource|, |underlyingSourceDict|, |highWaterMark|, |sizeAlgorithm|).
 </div>
 
+<div algorithm>
+ The static <dfn id="rs-from" method for="ReadableStream">from(|asyncIterable|)</dfn> method steps
+ are:
+
+ 1. Let |stream| be undefined.
+ 1. Let |iteratorRecord| be undefined.
+ 1. Let |startAlgorithm| be the following steps:
+  1. Set |iteratorRecord| to ? [$GetIterator$](|asyncIterable|, async).
```

Comment:
Putting this inside startAlgorithm, instead of just doing it straightaway, means that from() also works if [Symbol.asyncIterator]() returns a promise for an async iterator. That's probably not intended.